frontend/src/pages/GraphicalSelector/GraphicalSelector.tsx

```diff
@@ -200,6 +203,29 @@ const GraphicalSelector = () => {
     }
   };
 
+  const zoomIn = () => {
+    if (!graph) return;
+    const zoom = graph.getZoom();
+    let zoomRatio;
+    if (zoom >= 1) {
+      zoomRatio = 0.5;
+    } else {
+      zoomRatio = 0.1;
+    }
+    graph.zoomTo(zoom + zoomRatio);
+  };
+  const zoomOut = () => {
+    if (!graph) return;
+    let zoomRatio;
+    const zoom = graph.getZoom();
+    if (zoom >= 1) {
+      zoomRatio = 0.5;
```

Comment:
> yep zoomRatio is the new variable

Oh like I mean can we define `0.5` and `0.5` as const? i.e. example, fyi not sure the context of the var so rename these vars as suited
```
// place these var declaration at the top of the file (just after the imports)
// consider rename the variable to a better context. Not entirely sure what 0.5 and 0.1 means in this case.
const ZOOM_RATIO_IN = 0.5;
const ZOOM_RATIO_OUT = 0.1;
// might also make 1 as the zoom limit? again rename var - im just gussing what 1 means
const ZOOM_LIMIT = 1;
...
// within the zoom out function
if (zoom >= ZOOM_LIMIT) zoomRatio = ZOOM_RATIO_IN;
else zoomRatio = ZOOM_RATIO_OUT;
```

Reply:
yep so ive just adjusted that essentially what occurs is that 0.5 and 0.1 should be used because when the zoom is <1 is snaps into really far out/in when the zoom is in that range. Hence values from 0.2 - 1 are really sensitive compared to the zoom values of 2-5. I've left it as ZOOM_LIMIT but could be a better name like turning point or end point... idk